Remotion/ObjectBinding/Web/UI/Controls/BocListImplementation/EditableRowSupport/EditModeController.cs

```diff
@@ -209,6 +211,8 @@ public void EndRowEditMode (bool saveChanges, BocColumnDefinition[] columns)
       {
         var editedRow = GetEditedRow();
 
+        Assertion.DebugIsNotNull (_rows[0].GetDataSource(), "The data source of _rows[0] must not be null.");
```

Comment:
What's the pattern for DebugIsNotNull between using a verbose msg vs letting Rider insert a default `_rows[0].GetDataSource() != null` message?

Reply:
I thought the message should always be verbose in this codebase - I'd use the generated message in most cases.

Reply:
I think I picked this up in Data NRTs. For the not-null debug messages that are just there to enfore NRT semantics, the generated messages should be fine, sometimes with added context (e.g. "foo != null when bar is true"). The verbose message isn't any more expressive and right now, the generated debug message gives us more details on the error that the verbose message. Also, I think there's a bunch of Debug-assertions that don't even have messages => the generated one is good for these cases.